src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -1246,6 +1253,28 @@ private static TypeSymbol RefineAccessThroughType(LookupOptions options, TypeSym
         }
 
         /// <summary>
+        /// A symbol is accessible for referencing in a cref if it is in the same assembly as the reference
+        /// or the symbols's effective visibility is not private.
+        /// </summary>
+        private bool IsCrefAccessible(Symbol symbol)
+        {
+            return !IsEffectivelyPrivate(symbol) || symbol.ContainingAssembly == this.Compilation.Assembly;
+        }
+
+        private static bool IsEffectivelyPrivate(Symbol symbol)
+        {
+            for (Symbol s = symbol; (object)s != null; s = s.ContainingSymbol)
+            {
+                if (s.DeclaredAccessibility == Accessibility.Private)
```

Comment:
You can, but it's not pretty. The following reference should link to [`Dictionary<TKey, TValue>.ICollection<KeyValuePair<TKey, TValue>>.Contains`](https://msdn.microsoft.com/en-us/library/cc672218.aspx).

```xml
<see cref="M:System.Collections.Generic.Dictionary`2.System#Collections#Generic#ICollection{System.Collections.Generic.KeyValuePair{`0,`1}}#Contains(System.Collections.Generic.KeyValuePair{`0,`1})"/>
```

However, due to an issue in MTPS, this reference won't actually get resolved. MTPS expects the reference to have the following form:

```xml
<see cref="M:System.Collections.Generic.Dictionary`2.System#Collections#Generic#ICollection{T}#Contains(System.Collections.Generic.KeyValuePair{`0,`1})"/>
```

This means you could actually have two methods with identical signatures if a type explicitly implemented two different parameterizations of the same generic type, and that type declared a method which did not have any arguments. [Despite my best efforts](http://stackoverflow.com/q/22416181/138304) I was not able to locate a situation where this actually led to a presentation bug.